department-of-veterans-affairs/va.gov-cms
# Adding --status-url option to calls to yaml-tests so github commit status links go to the test logs.

## Description

See #1229. 

## Testing done
Coming. Needs to run on latest devshop and latest Yaml Tests.

## Screenshots


## QA steps

As a github user
Visit a PR.
Click "Show all Checks" if needed.
Click "Details". Once all the tasks are complete, this should take you directly to the DevShop task that ran the tests.

## Definition of Done
